apps/src/lib/ui/lti/sync/LtiSectionSyncDialog.tsx

```diff
@@ -191,6 +201,48 @@ export default function LtiSectionSyncDialog({
 
   const hideCloseButton = currentView === SubView.SPINNER;
 
+  const SectionRow = (id: string, section: LtiSection) => {
+    const sectionOwnerName = Object.entries(section.instructors).find(
+      ([id, instructor]) => instructor.isOwner
+    )?.[1]?.name;
```

Comment:
I think this was leftover accidentally from when I had `instructors` as an object instead of an array. I can get rid of the `Object.entries` and just do a `find`